src/minesweeper/infrastructure/controllers/startGame/startGame.controller.spec.ts

```diff
@@ -1,16 +1,107 @@
 import request from 'supertest';
 import { createServer } from 'http';
 import { requestHandler } from '@minesweeper/infrastructure/server/requestHandler';
+import { endpoints } from '@minesweeper/infrastructure/server/constants';
+import { GetQueryParamsErrorMessages } from '@minesweeper/infrastructure/server/utils/getQueryParams';
 
 describe('start game controller', () => {
-  it('should return correct data', async () => {
+  it('should response Bad Request Error if empty data is provided as query params', async () => {
     const server = createServer(requestHandler);
 
-    const response = await request(server).get('/start?bombs=1&rows=2&columns=2');
+    const response = await request(server).get(`${endpoints.game}`);
```

Comment:
you can type `response` and avoid this error

Reply:
https://www.npmjs.com/package/@types/supertest

Reply:
ready